api.go

```diff
@@ -854,6 +911,48 @@ func (lapi *Client) UpdateTaskStatus(id string, phase string, progress float64)
 	return nil
 }
 
+func (lapi *Client) ImportAsset(url string, name string) (*Asset, *Task, error) {
+	var (
+		requestUrl = fmt.Sprintf("%s/api/asset/import", lapi.chosenServer)
+		input      = &importAssetRequest{URL: url, Name: name}
+		output     ImportAssetResp
+	)
+	err := lapi.doRequest("POST", requestUrl, "import_task", "", input, &output)
+	if err != nil {
+		return nil, nil, err
+	}
+	glog.V(logs.DEBUG).Infof("Created import task id=%s assetId=%s status=%s type=%s", output.Task.ID, output.Asset.ID, output.Task.Status.Phase, output.Task.Type)
+	return &output.Asset, &output.Task, nil
+}
+
+func (lapi *Client) TranscodeAsset(assetId string, name string, profile Profile) (*Asset, *Task, error) {
+	var (
+		url    = fmt.Sprintf("%s/api/asset/transcode", lapi.chosenServer)
```

Comment:
This is the deprecated path for this API. Use `/api/asset/:id:/transcode` instead and remove the `AssetId` from the payload.